docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P0 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Tomcat, Wildfly, Spring Boot
+  * All Java LTS versions + latest Java version if non-LTS
+* Benchmarking
+  * Startup overhead benchmark
```

Comment:
This feels lower than P0, maybe P2.

Reply:
Not sure. P1 - can agree, P2 - probably not.

Reply:
kept this at P0 for now (well, now P0 is P1) since the next priority level is all about "contributor experience", but agree this is flexible